packages/katello/katello/service-wait

```diff
@@ -57,25 +53,13 @@ before_start() {
 # after start or restart
 after_start() {
   case "$SERVICE" in
-    tomcat|tomcat6)
-      # RHBZ 789288 - wait until data port is available
-      wait_for_url $TOMCAT_TEST_URL
-      ;;
     foreman)
```

Comment:
I wouldn't mind getting rid of this. With the above configured URL (`http://localhost:5500/foreman/api`) this is probably never going to work. We also don't use the foreman as a standalone service.